pkg/plugins/plugin_test.go

```diff
@@ -98,16 +104,18 @@ func TestInstallDoesNotCleanIfInstallFails(t *testing.T) {
 	plugin, _ := LookUpPlugin(context.Background(), config, fs, "appA")
 	err := plugin.Install(context.Background(), config, fs, "2.0.1", testServers.StripeServer.URL)
 	require.Nil(t, err)
-	fileExists, _ := afero.Exists(fs, "/plugins/appA/2.0.1/stripe-cli-app-a")
+	file := fmt.Sprintf("/plugins/appA/2.0.1/stripe-cli-app-a%s", GetBinaryExtension())
```

Comment:
should we make these file paths as constants?

Reply:
worth considering for sure. I think we should look into this change once we know what the permanent fix this windows issue will be.